app/components/Console.js

```diff
@@ -587,7 +581,48 @@ export default class Console extends Component {
             defaultChecked={connectionSettings.isRealTimeMode}
             onChange={this.onFieldChange}
           />
-        </Form.Field>
+        </Form.Field> </Tab.Pane> }, 
+      { menuItem: "Advanced", render: () => <Tab.Pane>
+        <div className={styles['description']}>
+          <strong>Only modify if you know what you doing.</strong>&nbsp;
+          OBS integration lets you select the dolphin instance to be shown
+          if the game is active and hidden if the game is inactive.
+          You must download the &nbsp;
+          <a href="https://github.com/Palakis/obs-websocket">OBS Websocket Plugin</a>&nbsp;
+          for this feature to work.
+        </div>
+        <br/>
```

Comment:
I usually tend to prefer using margin styles instead of `<br/>` elements but I also don't mind that much

Reply:
I figured out how to add a spacer for this section and then tab area as a whole so we should be good there.